tensorflow_probability/python/mcmc/diagnostic.py

```diff
@@ -492,6 +494,12 @@ def potential_scale_reduction(chains_states,
 def _potential_scale_reduction_single_state(state, independent_chain_ndims,
                                             split_chains, validate_args):
   """potential_scale_reduction for one single state `Tensor`."""
+  # casting integers to floats for floating-point division
+  # check to see if the `state` is a numpy object for the numpy test suite
+  if state.dtype is tf.int64 or state.dtype is np:
```

Comment:
?

Reply:
Maybe use dtype_util.as_numpy_dtype(state.dtype) is np.float64 as a generic check.

Reply:
Another oversight (meant to have np.int64 there)